src/com/hms/flexyosisoftconnector/Tag.java

```diff
@@ -87,28 +114,50 @@ public String getTagValue() {
       return Long.toString(tagControl.getTagValueAsLong());
    }
 
-   public void recordTagValue(String time)
+   /**
+    * Record tag value will be called after parsing the historical log values.
+    * Historical logs are added to internal queue through this function.
+    *
+    * @param time is the time in historical logs in epoch time seconds
+    * (Referred to as timeInt in the EBD generated file)
+    * @param value is the value in historical logs
+    */
+   public void recordTagValue(String time, String value)
    {
+      // convert time string holding a epoch time in seconds to timestamp
+      long epochSeconds = Long.parseLong(time);
+      long epochTime = epochSeconds*1000;
+      SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-ddTHH:mm:ss");
+      String timestamp = dateFormat.format(new Date(epochTime));
+
+      // Logger.LOG_INFO("time in recordVal is "+timestamp);
+      Logger.LOG_INFO(getTagName()+": revording new val: "+value+" time: "+timestamp);

```

Comment:
should revording be rewording?

Reply:
this should be changed to LOG_DEBUG or deleted, and i meant recording cause it is in the recordValue function